doc/install.md

```diff
@@ -263,4 +263,4 @@ $ docker run --volume $HOME/.bitcoin:/home/user/.bitcoin:ro \
              --rm -i -t electrs-app
 ```
 
-To access the server from outside Docker, add `-p 50001:50001 -p 4224:4224` but be aware of the security risks. Good practice is to group containers that needs access to the server inside the same Docker network and not expose the ports to the outside world.
+To access the server from outside Docker, add `-p 50001:50001 -p 4224:4224` but be aware of the security risks. Good practice is to group containers that needs access to the server inside the same Docker network and not expose the ports to the outside world.
```

Comment:
Please undo this diff (seems that an EOL is missing)

Reply:
Okay!